### ISSUE TITLE
v12 fail

### ISSUE THREAD

**Issue by mhosman** [MurhafSousli/ngx-progressbar#370]
It fails sometimes with error message:

"1 errors occurred during unsubscription:\n1) Error: NG0600: Writing to signals is not allowed in a `computed` or an `effect` by default. Use `allowSignalWrites` in the `CreateEffectOptions` to enable this inside effects."

The bar get stuck at 100% and is not disappearing.

**Comment by MurhafSousli** [comment]
Please fill the template issue properly, what angular version are you using? and add [ stackblitz reproduction](https://stackblitz.com/edit/ngx-progressbar)

**Comment by mhosman** [comment]
I'm using the latest Angular version: 18.2. Is an erratic behaviour and is not related to my code:

```
<ng-progress ngProgressHttp ngProgressRouter flat aria-progressbar-name="progressbar"
    color="'#e91e63'"></ng-progress>
```

I get that error in console after loading some stuff.

**Comment by MurhafSousli** [comment]
Make a reproduction please, I need to see the issue.

**Comment by mhosman** [comment]
Sorry, I don't have time right now. I'm traveling today and I be off for 2 weeks.

**Comment by softworckx** [comment]
No idea that it's possible to reproduce it in Stackblitz but I'm facing exact the same issue as @mhosman.
I just upgraded from Angular 17.3 to 18.2 and from ngx-progressbar v11 to v12. I followed the wiki to change my module and styling and I'm getting the same error. I suppose the issue is somewhere in the ngProgressHttp, because that's the only way I interact with the progressbar. I can't reproduce it 100% at the same point, but when I click around in my webapp the error occurs after a few API-calls. 

![image](https://github.com/user-attachments/assets/30b42c60-a1c1-4553-8f0e-8a144fb5dacb)

**Comment by MurhafSousli** [comment]
Please try with v12.0.1 and let me know!

**Comment by softworckx** [comment]
@MurhafSousli, I have given it a try and the reported issue is fixed. Thanks a lot, your new version (v12.0.1) works like a charm.

**Comment by benterris** [comment]
Hello @MurhafSousli, 
I think the issue has been reintroduced in `v12.0.2`. I can confirm this issue is fixed in `v12.0.1`, but I have the exact same issue as @softworckx in the `v12.0.2`. It might be related to the switch from `allowSignalWrites` to `untracked` !

**Comment by MurhafSousli** [comment]
@benterris It cannot be, the fact switching to `untracked` is the proper way to remove this error. but the error maybe caused by you're use of the effect some where in the app.

**Comment by benterris** [comment]
I don't think it comes from our end, here is a minimal reproduction on stackblitz : https://stackblitz.com/edit/stackblitz-starters-epe3b1?file=src%2Fapp%2Fhome.component.ts

It's a very basic app with a router, using `NgProgressRouter` and `NgProgressHttp`. I added a delay on the `OnInit` of one of the components, which triggers the issue. 

My guess is that `untracked` does not function properly with `rxjs`, or maybe that some `untracked` are missing on the `onComplete`.

**Comment by MurhafSousli** [comment]
This should be opened in a separate ticket

**Comment by MurhafSousli** [comment]
@benterris It's Angular bug https://github.com/angular/angular/issues/57757

**Comment by MurhafSousli** [comment]
@benterris I released a beta version v13.0.0-beta.0, This should fix the error and also allow using both http and router features in one progress bar, use `npm i ngx-progressbar@beta`.

let me know if this fixes your issue!
